src/patternfly/layouts/Grid/docs/code.md

```diff
@@ -8,7 +8,7 @@ A Grid layout framework including responsive columns, row spans and gutters. Bre
 | -- | -- | -- |
 | `.pf-l-grid` | `<div>` | Initializes the grid layout. |
 | `.pf-l-grid__item` | `<div>` | Explicitly sets a child of the grid. This class isn't necessary, but it is included to keep inline with BEM convention, and to provide an entity that will later be used for applying modifiers. |
-| `.pf-m-gutter` | `.pf-l-grid` | Adds grid gutters. |
+| `.pf-m-gutter{-[size]{-on[breakpoint]}}` | `.pf-l-grid` | Adds grid gutters. |
```

Comment:
I could see a case for wanting a certain gutter size on the x-axis and a different gutter size on the y-axis, do you think this is worth exploring?

Reply:
I do! I'd like to start here though. Maybe we can wait until someone requests this to begin that work. Or do you think we should go ahead and create an issue and put it on the back burner?

Reply:
Maybe we start here and then in the next breaking change period we could get it in? so creating an issue would be good!